Styles/index.css

```diff
@@ -0,0 +1,91 @@
+:root {
+    --responsive-image-width: 100vw;
+}
+
+.main-banner {
+    position: relative;
+}
+
+.image-container {
+    margin: 0 auto;
+    margin-bottom: 3em;
+    height: 60vh;
+    max-width: 100%;
+}
+
+.image {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: top;
+}
+
+.overlay-text-container {
+    position: absolute;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    /* rgba helped get the background opacity changed to 0.75, opacity property changed opacity of children too */
+    background-color: rgba(54, 54, 54, 0.75);
+    color: #fff;
```

Comment:
set variable for this color as you will use it many times

Reply:
Actually i have asked vikram a doubt regarding this in that thread, will do after he responds to that